src/engraving/rw/inoutdata.h

```diff
@@ -45,10 +45,11 @@ struct ReadInOutData {
     // for master - out
     // for except - in
     ReadLinks links;
-    std::optional<double> overridedSpatium = std::nullopt;
 
     // out
     SettingsCompat settingsCompat;
+    std::optional<double> overridedSpatium = std::nullopt;
```

Comment:
But this one did belong to "in". You provide the overrided spatium as an input, and collect the saved spatium as an output, right?

Also, I think these things could have better names. `Overrided` is incorrect English and should be `overridden`. And instead of `savedSpatium`, I would say `originalSpatium`.